.stylelintrc

```diff
@@ -1,15 +0,0 @@
-{
-  "processors": ["stylelint-processor-styled-components"],
-  "extends": [
-    "stylelint-config-standard",
-    "stylelint-config-styled-components"
-  ],
-  "syntax": "scss"
```

Comment:
There was a syntax error (no comma) here that prevented stylelint from parsing the configuration. Syntax errors are easier to see when the editor knows the file type, so I moved this to a JS file.